The `osx` CI job failed for the following error: 
~~~
-- Downloading https://jaist.dl.sourceforge.net/project/geographiclib/distrib/archive/GeographicLib-1.47-patch1.zip...
-- Downloading https://jaist.dl.sourceforge.net/project/geographiclib/distrib/archive/GeographicLib-1.47-patch1.zip... Failed. Status: 7;"Couldn't connect to server"
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:175 (message):
      
      Failed to download file.
      If you use a proxy, please set the HTTPS_PROXY and HTTP_PROXY environment
      variables to "https://user:password@your-proxy-ip-address:port/".
      Otherwise, please submit an issue at https://github.com/Microsoft/vcpkg/issues

Call Stack (most recent call first):
  ports/geographiclib/portfile.cmake:3 (vcpkg_download_distfile)
  scripts/ports.cmake:90 (include)
~~~
It seems a temporary network connection problem (I am able to save the file correctly from that URL) unrelated to this PR.

--- a/ports/ode/CONTROL
+++ b/ports/ode/CONTROL
@@ -1,4 +1,4 @@
 Source: ode
-Version: 0.16
+Version: 0.16.1
 Homepage: https://bitbucket.org/odedevs/ode/src/default/
 Description: Open Dynamics Engine

--- a/ports/ode/portfile.cmake
+++ b/ports/ode/portfile.cmake
@@ -1,10 +1,8 @@
-include(vcpkg_common_functions)
-
 vcpkg_from_bitbucket(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO odedevs/ode
-    REF 0.16 
-    SHA512 6a98882aa3e6267423f745ec48f2472d330f94fa395c459e116174093ef1d479368efc0514ef04eff4e62eb7c3520a7a544fc3ed66ff2f1bd06bc13df4110581
+    REF 0.16.1
+    SHA512 04429cae1b8fc703e53880c5de78293cee46fe4855c96ca7006bd5848255a0df004b75716a6b30ff5176df004e2bec29b2a31d4af8e7ac59da18f0af2eed8396
     HEAD_REF master
 )
 
@@ -15,7 +13,7 @@ vcpkg_configure_cmake(
 )
 
 vcpkg_install_cmake()
-vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/ode-0.16.0)
+vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/ode-0.16.1)
 
 if(VCPKG_LIBRARY_LINKAGE STREQUAL static)
     file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/bin ${CURRENT_PACKAGES_DIR}/debug/bin)

